<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	rgb(255, 0, 0)
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Text</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SIGN UP WINDOW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_thirdstatus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Text</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>[+]STATUS : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_backtologin">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK TO LOGIN</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>401</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="label_usename">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>UserName : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pass">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PassWord :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Email :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pwd">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SIGNUP</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
